repositories/user.js

```diff
@@ -1,17 +1,27 @@
-const User = require("../db/models/User")
-
+const User = require("../db/models/User");
+const jwt = require('jsonwebtoken');
```

Comment:
as far as I can see you resolve comment and do nothing 
<img width="1078" alt="image" src="https://github.com/becarselman/ant-colony-pharaoh-backend/assets/102960105/94a4572f-5a5c-4e50-8fbc-84e641e8c581">